clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -888,22 +888,26 @@ default ListConsumerGroupsResult listConsumerGroups() {
     }
 
     /**
-     * List the consumer group offsets available in the cluster.
-     *
+     * List the consumer group offsets available in the cluster for the given list of consumer
+     * groups.
+     * @param groupIds List of consumer group ids to list offsets for.
      * @param options The options to use when listing the consumer group offsets.
      * @return The ListGroupOffsetsResult
      */
-    ListConsumerGroupOffsetsResult listConsumerGroupOffsets(String groupId, ListConsumerGroupOffsetsOptions options);
+    ListConsumerGroupOffsetsResult listConsumerGroupOffsets(List<String> groupIds, ListConsumerGroupOffsetsOptions options);
```

Comment:
Don't we need to keep the existing methods for backward compatibility? We could perhaps deprecate them.